CHANGELOG.md

```diff
@@ -19,6 +19,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Only print on rank 0 to avoid duplicates of all print statements.
 [\#103](https://github.com/mllam/neural-lam/pull/103) @simonkamuk @sadamov
 
+### Maintenance
+- update npyfiles MEPS and MDP DANRA example dataset to use small overlapping
+  domain (~100x100 grid points) with overlapping time-range (first two weeks of
+  April 2022) [\#110](https://github.com/mllam/neural-lam/pull/110), @leifdenby
+
```

Comment:
```suggestion
```

This is not part of this PR.

Reply:
Yes, I was trying to sneak in a changelog for the change in #110 😳 but I will make a separate PR to make the missing changelog update.